supervisor/api/__init__.py

```diff
@@ -401,7 +401,7 @@ def _register_supervisor(self) -> None:
 
         async def get_supervisor_logs(*args, **kwargs):
             try:
-                return await self._api_host.advanced_logs(
+                return await self._api_host.advanced_logs_handler(
                     *args, identifier="hassio_supervisor", **kwargs
                 )
             except Exception as err:  # pylint: disable=broad-exception-caught
```

Comment:
Now that we have a known use case (as this PR got triggered by devcontainer usage, see #5021), can we handle this specific and make it not report to Sentry?

I guess the exact error here might depond on weather systemd-journal-gatewayd is available? E.g. would the exception be different if we were to revert https://github.com/home-assistant/devcontainer/pull/6?

Reply:
Let's get this handled in a follow up PR. This PR does what it advertises: Fix the fallback as it is intended currently.